lib/Shipment/UPS.pm

```diff
@@ -189,76 +242,62 @@ has 'carbon_neutral' => (
   default => undef,
 );
 
-=head2 saturday_delivery
+=head1 Type Maps
 
-Set the Saturday Delivery Indicator - the shipment must be delivered on a Saturday
+=head2 service_map
 
-type: Bool
+UPS returns service codes without descriptions. This is mapped here so that we can display 'UPS Ground' instead of '03'.
 
 =cut
 
-has 'saturday_delivery' => (
-  is => 'rw',
-  isa => Bool,
-  default => undef,
+my %service_map = (
```

Comment:
these local hashes need to be changed back to attributes - see commit d8e7dc2 and PR #35 for background